client/src/lib/components/sandbox/play/Arena.svelte

```diff
@@ -80,6 +227,7 @@
 		height={arenaHeight}
 		class="border border-slate-400 mx-auto"
 		on:mousemove={onMouseMove}
+		on:mousedown={onMouseDown}
```

Comment:
Idea for a separate component:

`<ArenaCanvas>` which is a wrapper for each phase canvas, which only knows how to draw itself, in the way that `<PhaseInput>` used to work.